src/Connections/SetupConnection.ts

```diff
@@ -1,7 +1,7 @@
 // We need to instantiate some functions which are not directly called, which confuses typescript.
```

Comment:
Could I get you to write a page similar to https://matrix-org.github.io/matrix-hookshot/latest/usage/room_configuration/github_repo.html so we have a record of how to setup a Jira connection via commands?

Reply:
Done!